Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -17,10 +17,19 @@ public static partial class Calendars
         const string weeklyFrequency = "WEEKLY";
         const string monthlyFrequency = "MONTHLY";
         const string yearlyFrequency = "YEARLY";
+        const string byFrequencySearch = "FREQ=";
+        const string byDaySearch = "BYDAY=";
+        const string byMonthDaySearch = "BYMONTHDAY=";
+        const string byMonthSearch = "BYMONTH=";
+        const string bySetPosSearch = "BYSETPOS=";
+        const string byIntervalSearch = "INTERVAL=";
+        const string byCountSearch = "COUNT=";
 
         static async Task<IEnumerable<Calendar>> PlatformGetCalendarsAsync()
         {
-            await Permissions.RequestAsync<Permissions.CalendarRead>();
+            await Task.Delay(0);
```

Comment:
I assume we need to remove these delays now that we've checked the permissions behaviour?

Reply:
Yep that should no longer exist, though the problem still persists, Unsure of underlying cause.